sdk/cosmos/azure-cosmos-table/pom.xml

```diff
@@ -0,0 +1,17 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+ http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.example</groupId>
```

Comment:
* The contents of this pom.xml should look similar to the template. It appears to be missing the distribution information and some other features.
  * https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/template/azure-sdk-template/pom.xml

* The groupId, artifactId, and versionId should match our guidelines (https://azure.github.io/azure-sdk/java_design.html#maven)

```xml
<groupId>com.azure</groupId>
<artifactId>azure-cosmos-table</artifactId> <!-- {x-version-update;com.azure:azure-cosmos-table;current} -->
<version>1.0.0-beta.1</version>
```

* You may notice the metadata `<!-- {x-version-update;com.azure:azure-cosmos-table;current} -->`, this is what we use to keep track of our dependencies. You'll have to add a line in this file with your library.
  * https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_client.txt#L21
  * In alphabetical order, add `com.azure:azure-cosmos-table;1.0.0-beta.1;1.0.0-beta.1`